src/main/java/net/dv8tion/jda/core/requests/WebSocketClient.java

```diff
@@ -288,46 +289,44 @@ else if (audioRequest != null)
                         switch (stage)
                         {
                             case RECONNECT:
-                                audioRequest.setStage(ConnectionStage.CONNECT);
                             case DISCONNECT:
                                 packet = newVoiceClose(audioRequest.getGuildIdLong());
                                 break;
                             default:
                             case CONNECT:
-                                //If we didn't get RateLimited, Next allowed connect request will be 2 seconds from now
-                                audioRequest.setNextAttemptEpoch(System.currentTimeMillis() + 2000);
                                 packet = newVoiceOpen(audioManager, channel);
                         }
                         needRatelimit = !send(packet.toString(), false);
                         if (!needRatelimit)
                         {
-                            //If the connection is already established, then the packet just sent
-                            // was a move channel packet, thus, it won't trigger the removal from
-                            // queuedAudioConnections in VoiceServerUpdateHandler because we won't receive
-                            // that event just for a move, so we remove it here after successfully sending.
-                            if (audioManager.isConnected() && stage == ConnectionStage.CONNECT
-                                || stage == ConnectionStage.DISCONNECT)
+                            //If we didn't get RateLimited, Next allowed connect request will be 2 seconds from now
+                            audioRequest.setNextAttemptEpoch(System.currentTimeMillis() + 2000);
```

Comment:
Will moving this here effect the Audio Resume system? @MinnDevelopment

Reply:
No audio resume does not interact with the mWS in any way. However this will heavily affect the RECONNECT system of audio connections meaning it will send DISCONNECT then wait 2 seconds and then eventually send CONNECT which I don't agree is good. I recommend reverting this change to only be in effect for CONNECT.